app/views/layouts/darkswarm.html.haml

```diff
@@ -2,7 +2,9 @@
   %head
     %meta{charset: 'utf-8'}/
     %meta{name: 'viewport', content: "width=device-width,initial-scale=1.0"}/
-
+    %meta{:content => yield(:title).empty? ? "#{t(:title)}" : yield(:title), :property => "og:title"}/
```

Comment:
* I would find it more readable to put the `property` key first instead of the `content` one. 
* Then you should adopt this hash syntax: `key: value` instead of `:key => value`